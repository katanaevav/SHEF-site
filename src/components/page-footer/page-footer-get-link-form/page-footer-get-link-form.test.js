import React from "react";
import renderer from "react-test-renderer";
import PageFooterGetLinkForm from "./page-footer-get-link-form.jsx";

it(`Render PageFooter`, () => {
  const tree = renderer.create(
      <PageFooterGetLinkForm />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
