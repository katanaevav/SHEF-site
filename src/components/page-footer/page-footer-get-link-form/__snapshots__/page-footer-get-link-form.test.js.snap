// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PageFooter 1`] = `
<form
  action="https://echo.htmlacademy.ru"
  className="section-connect__form"
  method="POST"
>
  <p
    className="section-connect__form-title"
  >
    Отправим вам ссылку на наше приложение
  </p>
  <div
    className="section-connect__form-input-wrapper"
  >
    <input
      className="section-connect__form-input-email"
      name="e-mail"
      placeholder="Email"
      required={true}
      type="email"
    />
    <input
      className="section-connect__form-button"
      type="submit"
      value="Получить ссылку"
    />
  </div>
  <div
    className="section-connect__form-input-wrapper"
  >
    <input
      className="section-connect__form-checkbox"
      id="footer-check-policy"
      name="accept-policy"
      type="checkbox"
      value="yes"
    />
    <label
      className="section-connect__form-checkbox-label"
      htmlFor="footer-check-policy"
    >
      согласен с условиями
      <a
        href="#"
      >
         Политики конфиденциальности
      </a>
    </label>
  </div>
</form>
`;
